sale_order_secondary_unit/views/sale_order_views.xml

```diff
@@ -12,20 +12,25 @@
                 expr="//field[@name='order_line']/form//field[@name='price_subtotal']"
                 position="after"
             >
-                <field
-                    name="secondary_uom_qty"
-                    class="oe_inline oe_no_button"
-                    attrs="{'readonly': [('state', 'in', ('done', 'cancel'))]}"
-                    optional="show"
-                />
-                <field
-                    name="secondary_uom_id"
-                    class="oe_inline"
-                    domain="[('product_tmpl_id.product_variant_ids', 'in', [product_id])]"
-                    attrs="{'readonly': [('state', 'in', ('done', 'cancel'))]}"
-                    optional="show"
-                />
-                <field name="secondary_uom_unit_price" readonly="1" optional="show" />
```

Comment:
@pedrobaeza and this - why keep the field in model if not used anywhere?

Reply:
I'm not sure. I hope @sergio-teruel can tell.

Reply:
This field was added in the form view, this view normally is not used to create sale lines. And I saw that is not defined on tree view.. but I do not remember de author of it, mybe he can tell us more about these need..

Reply:
But the field was readonly so I assume that was informational field intended for viewing? I know that sale order line form is rarely used, still I don't think it was worth removing it